dotcom-rendering/src/components/Discussion/CommentForm.tsx

```diff
@@ -304,9 +305,7 @@ export const CommentForm = ({
 		setBody('');
 		setShowPreview(false);
 		setIsActive(false);
-		if (setCommentBeingRepliedTo) {
-			setCommentBeingRepliedTo();
-		}
```

Comment:
This whole `resetForm` method can be lifted up once `error`, `info`, `body` and `showPreview` have been lifted up. And we will be able to not pass the setter methods for all these as props (e.g. the `setIsActive` prop that gets introduced in this PR).